package.json

```diff
@@ -9,11 +9,12 @@
     "dist"
   ],
   "scripts": {
-    "test": "jest",
-    "test:watch": "jest --watch",
+    "test": "node --experimental-vm-modules node_modules/jest/bin/jest.js",
+    "test:watch": "node --experimental-vm-modules node_modules/jest/bin/jest.js --watch",
```

Comment:
I think no need in  --experimental-vm-modules

Reply:
Without this flag we have next error on CI
https://github.com/iden3/js-iden3-auth/actions/runs/4196204422/jobs/7276847162#step:6:14